packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -346,7 +346,7 @@ export async function load_node({
 
 	// generate __data.json files when prerendering
 	if (shadow.body && state.prerender) {
-		const pathname = `${event.url.pathname}/__data.json`;
+		const pathname = `${event.url.pathname.replace(/\/$/, '')}/__data.json`;
```

Comment:
Are there other paths besides literally `'/'` that might end with a `/` that we have to worry about here? Can we check for the literal string `'/'` here which would (presumably) be slightly less computationally intensive?

Reply:
yeah, this is also fixing the `trailingSlash: 'always'` case

Reply:
Gotcha, okay, that makes me sufficiently disinterested in microoptimizations, then,